ui/components/Filters.js

```diff
@@ -1470,13 +1463,13 @@ const ImportModal = React.memo((props) => {
 });
 
 const PublishModal = React.memo((props) => {
-  const { publishFormSchema, handleClose, handlePublish, title } = props;
+  const { handleClose, handlePublish, title } = props;
 
   return (
     <Modal
       open={true}
-      schema={publishFormSchema.rjsfSchema}
-      uiSchema={publishFormSchema.uiSchema}
+      schema={publishCatalogItemSchema}
+      uiSchema={publishCatalogItemSchema}
```

Comment:
Why this change? you are getting issue because of this please revert this change or use below code it should fix your issue.

```suggestion
      schema={publishFormSchema.rjsfSchema}
      uiSchema={publishFormSchema.uiSchema}
```